### ISSUE TITLE
Auto generate startup form

### ISSUE THREAD

**Issue by ShoodSyr** [icsharpcode/CodeConverter#399]
"VB -> C#: "

### Input code
```vbnet
'Two errors:
' 1- in Application.Designer.vb
Protected Overrides Sub OnCreateMainForm()
            Me.MainForm = Global.WindowsApplication1.Form1
End Sub

'2- in Form1, cbo is ComboBox
Lblcolor.BackColor = Color.FromName(cbo.SelectedItem)
```

### Erroneous output
```C#
' 1- error message: 'Form1' is a type, which is not valid in the given context
protected override void OnCreateMainForm()
        {
            MainForm = Form1;
        }

' 2- cannot convert from 'object' to 'string'
Lblcolor.BackColor = Color.FromName(cbo.SelectedItem);
```

### Details
* VS extension on Visual Studio 2017
* Version in use: 7.0.0
Attached full project in vb.net and C# converted
[vb.net_2_c.zip](https://github.com/icsharpcode/CodeConverter/files/3776382/vb.net_2_c.zip)

**Comment by GrahamTheCoder** [comment]
Thanks for getting in touch. The first issue is a specific (extreme) case of #375 so we'll track progress there.

The second issue you mentioned actually has been fixed in master (it was #365) but not released yet, so it will be in version 7.3 in a couple of weeks (or you can install a dev build from the ci system or a local build if you'd like to try it before then)

Yourexample will be a great test case. I'll close this issue and track the progress on those linked ones (I might rename them so they're easier to find, apologies for that!).

**Comment by ShoodSyr** [comment]
Thank you for your reply. I'll wait for the next version.
However, your extension is very helpful and helped me a lot
Thank you very much
